cli/azd/internal/telemetry/appinsights-exporter/endpoint_config.go

```diff
@@ -0,0 +1,81 @@
+package appinsightsexporter
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+)
+
+const (
+	defaultIngestionPrefix = "dc"
+	defaultBaseEndpoint    = "https://dc.services.visualstudio.com"
+	defaultEndpointPath    = "v2/track"
+)
+
+const (
+	instrumentationKey_Setting = "InstrumentationKey"
+	endpointSuffix_Setting     = "EndpointSuffix"
+	ingestionEndpoint_Setting  = "IngestionEndpoint"
+)
+
+type EndpointConfig struct {
+	EndpointUrl        string
+	InstrumentationKey string
+}
+
+// NewEndpointConfig parses a connection string, returning the endpoint configuration from the connection string.
```

Comment:
Consider adding a comment pointing at https://learn.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string?tabs=net#schema for the schema?

Reply:
Yup, had the same thought after looking at it agian..